src/components/PersonPage/PersonPage.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import classNames from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+import { SortField } from '../../types/SortField';
+
+type Props = {
+  people: Person[];
+};
+
+export const PersonPage: React.FC<Props> = ({ people }) => {
+  const { personSlug } = useParams();
+
+  const getParentsLink = (personName: string | null) => {
+    if (!personName) {
+      return '-';
+    }
+
+    const parent = people.find(({ name }) => name === personName);
+
+    if (parent) {
+      return <PersonLink person={parent} />;
+    }
+
+    return personName;
+  };
```

Comment:
Better would be tp prepare your people array after API call and add `father` and `mother` to every person